worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -159,26 +157,42 @@ private boolean filterByYear(IDomainResource patient) {
      * @return patientId if present, null otherwise
      */
     private Identifiers extractPatientId(IDomainResource patient) {
+        List<PatientIdentifier> ids = IdentifierUtils.getIdentifiers(patient);
         // Get patient beneficiary id
         // if not found eobs cannot be looked up so do not return a meaningful list
-        String beneId = IdentifierUtils.getBeneId(patient);
-        if (beneId == null || beneId.isEmpty()) {
+        PatientIdentifier beneIdObj = IdentifierUtils.getBeneId(ids);
+        String beneId = null;
+        if (beneIdObj == null) {
             missingBeneId += 1;
             return null;
+        } else {
+            beneId = beneIdObj.getValue();
+            if (beneId == null || beneId.isEmpty()) {
+                missingBeneId += 1;
+                return null;
+            }
         }
```

Comment:
Would really benefit from a refactor to method as there are varying levels of abstraction.  This whole section can be replaced by
`        String beneId = extractBeneId(IdentifierUtils.getBeneId(ids));
`

Where the new method looks something like this:
```    
private String extractBeneId(PatientIdentifier beneIdObj) {
        if (beneIdObj == null) {
            missingBeneId += 1;
            return null;
        }

        String beneId = beneIdObj.getValue();
        if (beneId == null || beneId.isEmpty()) {
            missingBeneId += 1;
            return null;
        }

        return beneId;
    }
```